uber/decorators.py

```diff
@@ -279,8 +274,72 @@ def __call__(self, klass):
 
 
 register = template.Library()
+
+
 def tag(klass):
     @register.tag(klass.__name__)
     def tagged(parser, token):
         return klass(*token.split_contents()[1:])
     return klass
+
+
```

Comment:
In core Uber, they all just stay in ``uber/model_checks.py`` since that's a good place for them.  But we want event-specific plugins to be able to write their own validations.  Theoretically they could define their own methods and then dynamically jam them into ``uber.model_checks`` but that seems kludgy.